Pattern.pde

```diff
@@ -519,6 +519,146 @@ public float getPlanePointDist(PVector origin,PVector normal,PVector pos){
 }
 
 
+
+/** ************************************************************ PIXIE PATTERN
```

Comment:
Why is it in graph pattern? I totally missed that, because it doesnt seem like a graph specific pattern.
I suggest we remove it from graph pattern. thoughts?

Reply:
Got it, you are manually adding in the `lx.model` into each GraphPattern via inheritance. Clever.
I'm moving the new pattern in Graph Pattern as well.